libobs/obs-nix.c

```diff
@@ -813,7 +978,7 @@ static bool key_pressed(xcb_connection_t *connection,
 
 	} else if (key == OBS_KEY_META) {
 		pressed = keycode_pressed(reply, context->super_l_code) ||
-		          keycode_pressed(reply, context->super_r_code);
+			  keycode_pressed(reply, context->super_r_code);
```

Comment:
This appears to be another whitespace change that wasn't necessary/relevant.  Make sure to review your diffs in the future to make sure the code you've committed is relevant to the changes you're trying to submit.